src/server/journal/streamer.cc

```diff
@@ -122,7 +116,7 @@ bool RestoreStreamer::ShouldWrite(SlotId slot_id) const {
 }
 
 void RestoreStreamer::WriteBucket(PrimeTable::bucket_iterator it) {
-  DCHECK_LT(it.GetVersion(), snapshot_version_);
+  DCHECK_LE(it.GetVersion(), snapshot_version_);
```

Comment:
we should write bucket only if it.GetVersion() < snapshot_version_
also this check will fail as snapshot_version_ can be bigger than it.GetVersion()